automation/test.sh

```diff
@@ -27,6 +27,19 @@ readonly TEMPLATES_SERVER="gs://kubevirt-vm-images"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
 
+# Skip if it's docs changes only
+if [ ${CI} == "true" ]; then
+    SKIP_PATTERN="^(docs/.*|.*/(OWNERS|OWNERS_ALIASES|\.(md|txt)))$"
+    CI_GIT_ALL_CHANGES=$(git diff --name-only $PULL_BASE_SHA $PULL_PULL_SHA)
```

Comment:
`PULL_PULL_SHA` does not exist in batch jobs so we would have to handle the batch case too or else this could end up quietly skipping all the job in a batch.

Also adding a check to make sure `CI_GIT_ALL_CHANGES` is not empty would be good in case one of the SHA env variables is not present like in the batch case.

Reply:
Oh, good catch!

@fabiand FYI

Reply:
Sounds like we should not do this "skip" for batch runs, i.e. to condition it on the value existence of both env vars.

Reply:
I think the check will take care of that itself, since the change set reflects whether it only contains docs in the merged PR, no?

Reply:
FTR here's the docs for this: https://docs.prow.k8s.io/docs/jobs/#job-environment-variables